src/tests/integration/test-data/foundry-prove-all

```diff
@@ -0,0 +1,295 @@
+AccountParamsTest.testDealConcrete()
+AccountParamsTest.testDealSymbolic(uint256)
+AccountParamsTest.testEtchConcrete()
+AccountParamsTest.testEtchSymbolic(bytes)
+AccountParamsTest.testFail_GetNonce_false()
+AccountParamsTest.testFail_GetNonce_true()
+AccountParamsTest.test_GetNonce_false()
+AccountParamsTest.test_GetNonce_true()
+AccountParamsTest.test_getNonce_unknownSymbolic(address)
+AccountParamsTest.test_Nonce_ExistentAddress()
+AccountParamsTest.test_Nonce_NonExistentAddress()
+AccountParamsTest.testNonceSymbolic(uint64)
+AddrTest.test_addr_false()
+AddrTest.test_addr_symbolic(uint256)
+AddrTest.test_addr_true()
+AddrTest.test_builtInAddresses()
+AddrTest.testFail_addr_false()
+AddrTest.testFail_addr_true()
```

Comment:
Are we able to enumerate this list instead by inspecting the foundry kompiled output? That would be good for making sure that we don't accidentally not enable a newly added test.

Either that, or add a simple test-step which just compares this list against foundry compiled output to make sure that it's complete.